File: sample.py
class Person:
  def __init__(self, name, age):
    self.name = name;
    self.age = age
  def toString(self):
    return self.name + ", " + str(self.age)

person1 = Person("太郎", 14)
print(person1.toString())
print("私の名前は" + person1.name + "です。" + str(person1.age) + "歳です。")

File: sample.cs
using System;

class Person
{
	public string name;
	public int age;
	public Person(string name, int age)
	{
		this.name = name;
		this.age = age;
	}
	public string ToString()
	{
		return $"{this.name}, {this.age}";
	}
}

class MainProgram
{
	public static void Main(string[] args)
	{
		Person person1 = new Person("太郎", 14);
		Console.WriteLine(person1.ToString());
		Console.WriteLine($"私の名前は{person1.name}です。{person1.age}歳です。");
	}
}